File: samples/python/minidemo.py
#! /usr/bin/env python

import urllib2
import cv

cap = highgui.cv.CreateFileCapture("../c/tree.avi")
img = highgui.cv.QueryFrame(cap)
print "Got frame of dimensions (", img.width, " x ", img.height, ")"

highgui.cv.NamedWindow("win", highgui.cv.CV_WINDOW_AUTOSIZE)
highgui.cv.ShowImage("win", img)
highgui.cv.MoveWindow("win", 200, 200)
highgui.cv.WaitKey(0)

File: samples/octave/minidemo.m
#! /usr/bin/env octave -q
highgui

highgui.cvNamedWindow("win", highgui.CV_WINDOW_AUTOSIZE)
cap = highgui.cvCaptureFromFile("/home/asbach/Source/ObjectDetection/older/avi/table.avi")
img = highgui.cvQueryFrame(cap)

img
printf("Got frame of dimensions (%i x %i)",img.width,img.height);

highgui.cvShowImage("win", img)
highgui.cvMoveWindow("win", 200, 200)
highgui.cvWaitKey(0)

pilimg = opencv.Ipl2PIL(img)
pilimg

